File: 프로그래머스/lv2/43165. 타겟 넘버/타겟 넘버.py
def solution(numbers, target):
    cases = [0];
    for i in numbers:
        temp = [];
        for j in cases:
            temp.append(j + i);
            temp.append(j - i);
        cases = temp;
    count = 0;
    for i in cases: 
        if i == target: count += 1;
    return count

File: 프로그래머스/lv2/43165. 타겟 넘버/타겟 넘버.js
function solution(numbers, target) {
    let cases = [0];
    for(let i of numbers){
        const temp = [];
        for(let j of cases){
            temp.push(j + i);
            temp.push(j - i);
        }
        cases = temp;
    }
    let count = 0;
    for(let i of cases){
        if(i === target) count++;
    }
    return count
}